.github/workflows/release.yml

```diff
@@ -180,23 +196,59 @@ jobs:
       - name: Upload artifacts for ${{ matrix.os_type }} platform
```

Comment:
This should be on release only?

Reply:
The whole job is

Reply:
It's not

Reply:
ah, I got it confused with the other similar step, fixed